app/controllers/ticket_controller.rb

```diff
@@ -2,25 +2,28 @@ class TicketController < ApplicationController
   before_action :set_ticket, only: []
 
   def index     
-
     @tickets = Ticket.where(ticket_desk_id: params[:ticket_desk_id]).where(cinema_hall_id: params[:cinema_hall_id])    
     render json: @tickets
   end
 
   def show
     render json:[]
   end
+  
+  def new
```

Comment:
`new` action is not needed, cause it's related to Rails views. Please work only with `create` action